src/app/sidebar/sidebar.component.html

```diff
@@ -30,5 +30,5 @@
             </div>
```

Comment:
Can you change `mouseover` to `mouseenter` so the event doesn't trigger multiple times (when you mouse over a child element)?